packages/runtime/test/src/parser/parsing.ts

```diff
@@ -84,6 +84,39 @@ describe("Get Encoding for Byte Data", () => {
   });
 });
 
+describe("assertNumber", function () {
+  it("should pass with hex and dec", () => {
+    const hexValue = "0xaa3C3";
+    assert.deepEqual(assertNumber(hexValue, 1), hexValue);
+    const decValue = "343434";
+    assert.deepEqual(assertNumber(decValue, 1), decValue);
```

Comment:
don't need to use `deepEqual`. Simple equal is perfect here

Reply:
Thanks!